pkgs/development/python-modules/primp/default.nix

```diff
@@ -12,33 +12,71 @@
   patchelf,
   gcc-unwrapped,
   python,
+  fetchpatch,
 }:
 
 let
+  boringsslPatched = boringssl.overrideAttrs (oa: {
+    # boringssl source obtained from https://github.com/0x676e67/boring2/tree/1a0f1cd24e728aac100df68027c820f858199224/boring-sys/deps
+    src = fetchFromGitHub {
+      owner = "google";
+      repo = "boringssl";
+      rev = "44b3df6f03d85c901767250329c571db405122d5";
+      hash = "sha256-REELo7X9aFy2OHjubYLO1UQXLTgekD4QFd2vyFthIrg=";
+    };
+    modRoot = "./src";
+    patches = [
+      # A patch required to build boringssl compatible with `boring-sys2`.
+      # See https://github.com/0x676e67/boring2/blob/1a0f1cd24e728aac100df68027c820f858199224/boring-sys/build/main.rs#L486-L489
+      (fetchpatch {
+        name = "boringssl-44b3df6f03d85c901767250329c571db405122d5.patch";
+        url = "https://raw.githubusercontent.com/0x676e67/boring2/4edbff8cade24d5d83cc372c4502b59c5192b5a1/boring-sys/patches/boringssl-44b3df6f03d85c901767250329c571db405122d5.patch";
+        hash = "sha256-lM+2lLvfDHnxLl+OgZ6R8Y4Z6JfA9AiDqboT1mbxmao=";
+      })
+    ];
```

Comment:
Why are we pulling a two-year-old boringssl version?
Who monitors it for required security patches?

Guys, that's not cool.